src/content/docs/network-performance-monitoring/get-started/network-monitoring-guidelines.mdx

```diff
@@ -0,0 +1,330 @@
+
+### Compute scale [#compute-scale]
+
+The individual containers deployed with the KTranslate image are usually hosted on very small hosts and have minimal requirements as outlined in [our documentation](/docs/network-performance-monitoring/advanced/ktranslate-container-management/#container-requirements). However scaling this host’s CPU may be required in heavy polling scenarios. The primary motivator for scaling to a larger CPU footprint for a container is the amount of load presented to the task. In these situations it is normally better to run multiple containers to load balance instead of increasing the total size of your Docker host, which has underlying cost implications.
+
+### Summary [#summary]
+
+In the [common architecture example](#common-architecture) above, the following architectural considerations are in place:
+
+* Each geographical location has its own local containers used to collect and forward data to New Relic.
+* DC_01 has three containers:
+
+    * Container 1: SNMP polling
+    * Container 2: NetFlow v5 collection
+    * Container 3: Syslog collection
+
+    <Callout variant="tip" title="Recommendation">
+    In DC_01 there is a Class B private subnet (/16). Care needs to be taken to split up discovery targets into manageable subnet sizes to ensure the job has time to complete.
+    </Callout>
+
+* OFFICE_01 uses a single container for SNMP polling with discovery against a /24 subnet.
+
+* DC_02 has three containers, all dedicated to different templates in network flow collection:
+
+    * Container 1: NetFlow v9
+    * Container 2: IPFIX
+    * Container 3: sFlow
+
+    <Callout variant="tip" title="Recommendation">
+    In DC_02, there is an even larger Class A private subnet (/8); however, since there is no need for a discovery in this location there are no concerns about scaling the jobs. Depending on flows-per-second, there may be a need in the future to scale out into additional containers. In a situation like this, you simply need to configure your network flow exporters to target different ports on the Docker host.
+    </Callout>
+
+The guidance below will reference this sample architecture.
+
+## Maintaining your deployment [#maintaining-deployment]
+
+After initial installation, there are various techniques that are used to maintain the network monitoring observability footprint. Integrating configuration file changes with tools like Ansible are common, as is building GitOps pipelines around the architecture to support versioning and “guest” options where external teams can submit changes for review.
+
+The most common need for ongoing maintenance is in keeping the list of target devices accurate. There are three main discovery methods to update target devices: 
```

Comment:
@thezackm @cah-zack-mutchler Is it accurate to call these "discovery methods" (vs. "processes")?